.github/workflows/ci.yml

```diff
@@ -218,10 +218,13 @@ jobs:
             hashFiles('setup.py', 'requirements_test.txt', 'requirements.txt', 'pyproject.toml', 'setup.cfg', 'uv.lock') }}
       - name: Download all coverage artifacts
         uses: actions/download-artifact@v7
+        with:
+          pattern: coverage-*
```

Comment:
This `pattern` change isn't really relevant for this PR, but I've included it anyway.
We only upload coverage artifacts at the moment, but I think it's nice to make sure we explicitly only download coverage artifacts here, and nothing else. E.g. `coverage-3.11.14` and `coverage-3.12.12`